_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/scripts/@rhd/rhdp-search/rhdp-search-result.ts

```diff
@@ -159,28 +158,30 @@ export default class RHDPSearchResult extends HTMLElement {
         };
         this.kind = map[kind] || 'Web Page';
     }
+
     computeCreated(result) {
         this.created = result.fields.sys_created && result.fields.sys_created.length > 0 ? result.fields.sys_created[0] : '';
     }
+
     computeDescription(result) {
         var description = '';
         if(result.highlight && result.highlight.sys_description) {
             description = result.highlight.sys_description[0];
         } else if( result.highlight && result.highlight.sys_content_plaintext) {
-            description = result.highlight.sys_content_plaintext[0];               
+            description = result.highlight.sys_content_plaintext[0];
         } else if (result.fields && result.fields.sys_description) {
             description = result.fields.sys_description[0];
-        } else {
+        } else if (result.fields && result.fields.sys_content_plaintext) {
```

Comment:
This fixes the JS error that is preventing all results from showing.  If there is no description, the result item will still display without a description.